1.Bash-Scripts/ping_latency_report

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Get the latency report after ping to some host
+
+host="www.google.com"
+
+latency=$(ping -c 1 $host | awk -F'/' 'END {print $5}')
```

Comment:
This is good and okay.

You could have also matched for `time=..` from the `ping` output.
```
64 bytes from bud02s34-in-f14.1e100.net (142.250.180.238): icmp_seq=1 ttl=63 time=34.0 ms
```
like here, `time=34.0 ms`.